nata/simpleBlogHomePage/styles.css

```
@@ -0,0 +1,86 @@
+html {
+    margin: 0 auto;
+}
+
+header {
+    margin-left: 148px;
+    margin-right: 148px;
+}
+
+nav {
+    float: right
+}
+
+.large {
+    position: absolute;
```

Bad idea to make the block absolutely positioned and you don't need it here
```suggestion
    position: relative;
```